### PR TITLE
Move prefetch devs out of the larger build team

### PR DESCRIPTION
**Pull request by chmeliik**
They generally don't maintain anything in the build pipeline except for
the prefetch task.

Also: Don't run e2e-tests just for OWNERS changes (note that the e2e-tests _will_ trigger on this PR because it changes a file in `.tekton`)

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -9,9 +9,6 @@ aliases:
     - tisutisu
     - tnevrlka
     - MartinBasti
```
Comment (chmeliik) [inline OWNERS_ALIASES:11]: @MartinBasti do you want to be moved just to prefetch as well?

**Comment by arewm** [comment]
Added. I didn't add @eskultety to any team. We can do that too if desired.

**Comment by chmeliik** [comment]
> Added. I didn't add @eskultety to any team. We can do that too if desired.

Good point, https://github.com/orgs/konflux-ci/teams/build-maintainers please :pray:

File: OWNERS_ALIASES (hunk 1 above)
Comment (MartinBasti) [reply OWNERS_ALIASES:11]: proabably yes, eventually I'll be thrown there next year

Comment (MartinBasti) [reply OWNERS_ALIASES:11]: I meant to keep me :D (is Friday already?)

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**